enhancements/observability/multicluster-observability-addon-interoperability.md

```diff
@@ -0,0 +1,212 @@
+---
+title: multicluster-logs-traces-forwarding
+authors:
+  - "@pavolloffay"
+reviewers:
+  - "@periklis"
+  - "@alanconway"
+  - "@jcantrill"
+  - "@berenss"
+  - "@bjoydeep"
+approvers:
+  - "@periklis"
+  - "@alanconway"
+  - "@jcantrill"
+api-approvers:
+  - "@periklis"
+  - "@alanconway"
+  - "@jcantrill"
+creation-date: 2024-06-08
+last-updated: 2024-06-08
+tracking-link:
+  -
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+
+# Multi-Cluster telemetry data exportation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The objective of multi cluster observability is to offer users a capability to collect 
```

Comment:
`cluster observability addon` or `cluster observability operator`?

Reply:
MCO -multi cluster observability, which MCOA is part of

Reply:
Yes what I meant is that the sentence sounds like incomplete because it is not clear if you refer to one or another